src/main/scala/com/yotpo/metorikku/configuration/job/output/Hudi.scala

```diff
@@ -0,0 +1,16 @@
+package com.yotpo.metorikku.configuration.job.output
+
+case class Hudi(dir: String,
+                parallelism: Option[String],
+                maxFileSize: Option[String],
+                operation: Option[String],
```

Comment:
what are the default values for all this variables if I define only the dir ? copy on write, upsert, etc.... maybe add something about this on the readme